CHANGELOG.md

```diff
@@ -29,6 +29,18 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 The changelog is generated during the release process using [Github Changelog Generator](https://github.com/github-changelog-generator/github-changelog-generator) since `v9.1.1` release.
 
+## [v10.0.2](https://github.com/shopsys/shopsys/compare/v10.0.1...v10.0.2) (2022-07-20)
+
+:bug: Bug Fixes
```

Comment:
for me, only the first one is really a bug fix, I would maybe move the others into another section, maybe "Removed deprecations"? :thinking:

Reply:
🤔  sections are generated from the labels on Github (that reminds me we should be more strict about adding them)... 
maybe DX&Refactoring?

Reply:
chenged to dx and refactoring after discussion